runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -96,12 +97,18 @@
 	private static final String SUPER_ANTIFIRE_DRINK_MESSAGE = "You drink some of your super antifire potion";
 	private static final String SUPER_ANTIFIRE_EXPIRED_MESSAGE = "<col=7f007f>Your super antifire potion has expired.</col>";
 	private static final String SUPER_ANTIVENOM_DRINK_MESSAGE = "You drink some of your super antivenom potion";
+	private static final ImmutableList<Integer> HOME_TELEPORT_ENDING_ANIMATIONS = ImmutableList.of(
```

Comment:
Just Collection<> is fine for the type declaration

Reply:
Declared as `Collection`